docs/ops/activation/HSigmoid_5.md

```diff
@@ -0,0 +1,47 @@
+## HSigmoid<a name="HSigmoid"></a> {#openvino_docs_ops_activation_HSigmoid_5}
+
+**Versioned name**: *HSigmoid-5*
```

Comment:
Please rename to `HardSigmoid`

Reply:
`H` letter in `HSigmoid` also stands for Hard but it seems to be quite different operation.
It would be great to add info that `HSigmoid` and `HardSigmoid` are not equivalent

Reply:
`HSigmoid` seems a particular case of `HardSigmoid` but not an equivalent .